plugins/core/qPCV/PCVCommand.cpp

```diff
@@ -0,0 +1,101 @@
+#include "PCVCommand.h"
+#include "PCV.h"
+#include "qPCV.h"
+
+//qCC_db
+#include <ccGenericMesh.h>
+#include <ccHObjectCaster.h>
+#include <ccPointCloud.h>
+#include <ccProgressDialog.h>
+
+static const char COMMAND_PCV[] = "PCV";
+static const char COMMAND_PCV_N_RAYS[] = "N_RAYS";
+static const char COMMAND_PCV_IS_CLOSED[] = "IS_CLOSED";
+static const char COMMAND_PCV_180[] = "180";
+static const char COMMAND_PCV_RESOLUTION[] = "RESOLUTION";
```

Comment:
In C++11, these are normally `constexpr`. So:

```cpp
constexpr char COMMAND_PCV_RESOLUTION[] = "RESOLUTION";
```

Reply:
Addressed in d0da40e306c750c901df52c65bdc1aecdc79e88e